extensions/poetry/PhylumExt.toml

```diff
@@ -4,5 +4,5 @@ entry_point = "main.ts"
 
 [permissions]
 run = ["poetry"]
-read = ["./poetry.lock", "./pyproject.toml"]
-write = ["./poetry.lock", "./pyproject.toml"]
+read = ["~/.cache/pypoetry", "~/.local/lib", "./"]
```

Comment:
Also want to mention filesystem dependencies here, doubt this suffices for them.